app/config.py

```
@@ -41,6 +43,8 @@ def clowder_config(self):
         import app_common_python
 
         cfg = app_common_python.LoadedConfig
+        kessel_relations_hostname = DependencyEndpoints["kessel-relations"]["api"].hostname
```

The dependency might not even be there, shall we defend against it? Maybe at least throw a friendly error message ;)